src/App.js

```diff
@@ -1,24 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
+import Registration from "./pages/Registration/Registration";
```

Comment:
Don't create a Pages folder, should be a container.

Reply:
Done